graphdatascience/graph/graph_proc_runner.py

```diff
@@ -254,6 +256,7 @@ def streamNodeProperties(
         separate_property_columns: bool = False,
         **config: Any,
     ) -> DataFrame:
+        warning("Deprecated in favor of `gds.nodeProperties.stream`")
```

Comment:
This is not true for all server versions right? It's also interesting to think about how we warn about things like this, since the server also does it. If we do keep it, I'd rather have a decorator for it

Reply:
Check the server GraphStreamRelationshipPropertiesProc.java, I think it has been marked deprecated in 2.2 & 2.3? Server logs the warning in neo4j.log.
With a decorator it will be a `@deprecated` like `@client_only_endpoint`?